x-pack/plugins/security_solution/public/management/pages/policy/store/policy_list/services/ingest.ts

```diff
@@ -113,3 +115,20 @@ export const sendGetFleetAgentStatusForConfig = (
     },
   });
 };
+
+/**
+ * Get Endpoint Security Package information
+ */
+export const sendGetEndpointSecurityPackage = async (
```

Comment:
If we end up implementing #66777, then we can refactor this here to use that new API instead